### PR TITLE
add user info to leaderboard

### PR DESCRIPTION
**Pull request by bellatoris**

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #151** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/151" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/151?utm_source=stack-comment).

**Review by Zeniuus — APPROVED** [review]

File: subprojects/bounded_context/accessibility/infra/src/main/kotlin/club/staircrusher/accessibility/infra/adapter/in/security/AccessibilityBoundedContextSecurityConfig.kt [hunk 1]
```diff
@@ -3,12 +3,16 @@ package club.staircrusher.accessibility.infra.adapter.`in`.security
 import club.staircrusher.spring_web.security.SccSecurityConfig
 import club.staircrusher.stdlib.di.annotation.Component
 import org.springframework.security.web.util.matcher.AntPathRequestMatcher
+import org.springframework.web.bind.annotation.PostMapping
 
 @Component
 class AccessibilityBoundedContextSecurityConfig : SccSecurityConfig {
+    // TODO: 뭔가 불편하고 error prone, annotation으로 해결할 수 없을까?
```
Comment (Zeniuus) [inline subprojects/bounded_context/accessibility/infra/src/main/kotlin/club/staircrusher/accessibility/infra/adapter/in/security/AccessibilityBoundedContextSecurityConfig.kt:10]: 굳

File: subprojects/bounded_context/accessibility/application/src/main/kotlin/club/staircrusher/accessibility/application/port/in/AccessibilityApplicationService.kt [hunk 2]
```diff
@@ -41,17 +41,27 @@ class AccessibilityApplicationService(
...
+        val placeAccessibilityComments: List<WithUserInfo<PlaceAccessibilityComment>>,
+        val hasOtherPlacesToRegisterInSameBuilding: Boolean,
+        val isLastPlaceAccessibilityInBuilding: Boolean,
+    ) {
+         data class BuildingAccessibilityUpvoteInfo(
+             val isUpvoted: Boolean,
+             val totalUpvoteCount: Int,
+         )
+    }
+
```
Comment (bellatoris) [inline subprojects/bounded_context/accessibility/application/src/main/kotlin/club/staircrusher/accessibility/application/port/in/AccessibilityApplicationService.kt:64]: 이게 왜들어가있지

**Comment by bellatoris** [comment]
@bellatoris queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/151).

**Comment by bellatoris** [comment]
@bellatoris merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/151).

**Pull request merged**